src/main/docker/gitlab-jenkins-mysql.yml

```diff
@@ -41,9 +41,9 @@ services:
         environment:
             - MYSQL_ALLOW_EMPTY_PASSWORD=yes
             - MYSQL_DATABASE=Artemis
-        image: mysql:8.0.27
+        image: mysql:8.0.28-oracle
```

Comment:
Why do we have a MySQL container in this configuration anyway? To my understanding neither gitlab nor Jenkins use MySQL right?